src/vue/backend/src/App.vue

```diff
@@ -1,12 +1,131 @@
 <template>
-  <idx-block className="app">
-    <idx-block className="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
+    <idx-block :className="{app: isActive, 'nav-is-collapsed': navCollapsed}">
+        <Header>
+            <template v-slot:toggle>
+                <idx-button
+                    @click="toggleCollapse()"
+                    customClass="toggle-nav"
+                    :aria-expanded="navCollapsed ? 'false' : 'true'">
+                    <img src="@/assets/plus.svg" height="30" alt="IDX Broker" loading="lazy">
+                    Toggle</idx-button>
+            </template>
+        </Header>
+        <Navigation>
+            <idx-block className="nav">
+                <router-link to="/settings">
+                    <idx-block tag="span" className="link-icon">(icon)</idx-block>
+                    <idx-block tag="span" className="link-text">Settings</idx-block>
```

Comment:
If this class is not meant to be general for all link text, then I'd tie it to the above block and call it `nav__link-text`, same with the `link-icon` class above. If it is meant to be general, no changes needed